### PR TITLE
Clarify recoverable testing for kernel updates

### PR DESCRIPTION
**Pull request by wehimwich**
and a few more typos fixed

### CODE REVIEW

File: raid.adoc [hunk 1]
```diff
@@ -130,18 +130,58 @@ described in the <<Recover from a shelf disk>> section below.
...
 . If a rotation hasn't just been completed, perform one (as an extra backup)
 . Before proceeding verify that there is no recovery in progress, check with *mdstat*
 . Drop the *primary* disk out of the RAID (as a backup) by running *drop_primary*
 . Install and test the update
++
+
+[TIP]
+====
+
+If you are testing a kernel update, you must to take extra steps
```
Comment (wehimwich) [inline raid.adoc]: Typo here 'to'

File: raid.adoc [hunk 2]
```diff
@@ -130,18 +130,58 @@ described in the <<Recover from a shelf disk>> section below.
...
 If the update is deemed *successful*:
 
 [start=5]
 . Run *recover_raid* to add the *primary* disk back into the RAID.
++
+
+[TIP]
+====
+
+If you are testing a kernel update, you must to turn on the disk
```
Comment (wehimwich) [inline raid.adoc]: Again: 'to'

File: raid.adoc [hunk 3]
```diff
@@ -130,18 +130,58 @@ described in the <<Recover from a shelf disk>> section below.
...
+
+====
+
 . Once the recovery completes (this may only take a few minutes), the system has been successfully updated.
++
+
+[TIP]
+====
+
+If you are testing a kernel update, you must to reboot after the
```
Comment (wehimwich) [inline raid.adoc]: Again: 'to'

**Review by jfhquick — CHANGES_REQUESTED** [review]
So if we remove the 'tutu' (or rather the three to's) that you noted, then this is good to go.

**Pull request merged**